server/src/__tests__/controller/workspace.test.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable */
```

Comment:
eslint disable ?

Reply:
Test files shows eslint issues. Like:  describe/test function are not declared.